couchdb/server/compiler.py

```diff
@@ -78,3 +96,48 @@ def helper():
         'parent': mod,
         'id': (idx is not None) and (idx + '/' + name) or name
     })
+
+
+def import_b64egg(b64str, egg_cache=None):
+    """Imports top level namespace from base64 encoded egg file.
+
+    For Python 2.4 `setuptools <http://pypi.python.org/pypi/setuptools>`_
+    package required.
+
+    :param b64str: Base64 encoded egg file.
+    :type b64str: str
+
+    :return: Egg top level namespace or None if egg import disabled.
+    :rtype: dict
+    """
+    if iter_modules is None:
+        raise ImportError('No tools available to work with eggs.'
+                          ' Probably, setuptools package could solve'
+                          ' this problem.')
```

Comment:
Hmm... same as previous note. The function `pkgutil.iter_modules` does exists on my py26. But ... not sure why [documentation](https://docs.python.org/2.6/library/pkgutil.html) do not metion it.

Patch v2:
```diff
diff --git a/couchdb/server/compiler.py b/couchdb/server/compiler.py
index 119766b..b5e34d7 100644
--- a/couchdb/server/compiler.py
+++ b/couchdb/server/compiler.py
@@ -163,19 +163,12 @@ def resolve_module(names, mod, root=None):
 def import_b64egg(b64str, egg_cache=None):
     """Imports top level namespace from base64 encoded egg file.
 
-    For Python 2.4 `setuptools <http://pypi.python.org/pypi/setuptools>`_
-    package required.
-
     :param b64str: Base64 encoded egg file.
     :type b64str: str
 
     :return: Egg top level namespace or None if egg import disabled.
     :rtype: dict
     """
-    if iter_modules is None:
-        raise ImportError('No tools available to work with eggs.'
-                          ' Probably, setuptools package could solve'
-                          ' this problem.')
     egg = None
     egg_path = None
     egg_cache = (egg_cache or
```